dimod/generators/__init__.py

```diff
@@ -25,6 +25,7 @@
 from dimod.generators.magic_square import *
 from dimod.generators.multi_knapsack import *
 from dimod.generators.quadratic_assignment import *
+from dimod.generators.quadratic_knapsack import *
```

Comment:
IMO this can live under `knapsack` as well.

Reply:
Sounds good, should I combine the tests under one test class or separate them?

Reply:
I think combined makes sense to me. But no strong feelings either way.

Reply:
Updated: https://github.com/dwavesystems/dimod/pull/1391/commits/8ea7853e41cd72ce3743345204ffc5f09534b238